### ISSUE TITLE
Ongoing anime always empty

### ISSUE THREAD

**Issue by singhayushh** [ProjectSakura/Mirai#10]
### Issue type
Bug Report

### About the issue
The section under ongoing anime is always empty even though data is being passed from the backend to the frontend.
This issue is most likely caused because the response is being sent before scraping is totally completed leading to only partial content being actually passed.

Screen shot for reference:

![image](https://user-images.githubusercontent.com/52620158/111221485-8ede7e00-8600-11eb-8fa7-93aeda8a46cb.png)

I would like to take up this issue myself!

**Comment by ArunTeltia** [comment]
Yes I have seen this issue I am not really sure why this is happening, We have to check the route of ongoing anime for it on development environment sometimes its show anime sometimes it does not, i think this has something to do with the scraping time but i am not really sure about it.
